src/main/java/ava/Ui.java

```diff
@@ -0,0 +1,64 @@
+package ava;
+
+import ava.processor.TaskList;
+import ava.task.Task;
+
+/**
+ * Class to represent the user interfaces.
+ */
+public class Ui {
+    /**
+     * Shows exit message at the end of the program.
+     *
+     * @return Message printed when task is added.
+     */
+    public String printExit() {
+        return "Bye. Good luck on your tasks!\n *shutting down.....*";
+    }
+
+    /**
+     * Shows the added task.
+     *
+     * @param tasks TaskList.
+     * @param numOfTask Number of Tasks.
+     * @return Message printed when task is added.
+     */
+    public String showAddOnTask(TaskList tasks, int numOfTask) {
```

Comment:
I think that since this method returns a `String`, would it be better if the method was named `getAddOnTaskMessage` instead?